panc/src/main/scripts/panlint

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# coding=utf8
+
+import re
+import argparse
+from glob import glob
+from colorama import Fore, Style, init as colorama_init
+from sys import stdout, exit as sys_exit
+from inspect import getmembers, ismethod
+from prettytable import PrettyTable
+
+RS_COMMENT = r'(?:#.*|@{.*})$'
```

Comment:
i would also allow `\s*` for the `@{}` part (and possibly `@\w*\{`, to allow `@documentation{` sort of comments)